x-pack/plugins/apm/public/components/shared/links/apm/transaction_detail_link.tsx

```diff
@@ -68,5 +77,27 @@ export function TransactionDetailLink({
     search: location.search,
   });
 
-  return <EuiLink href={href} {...rest} />;
+  if (transactionName !== txGroupsDroppedBucketName) {
+    return <EuiLink href={href} {...rest} />;
+  }
+
+  return (
+    <EuiToolTip
+      content={i18n.translate('xpack.apm.transactionDetail.tooltip.message', {
+        defaultMessage:
+          "The transaction group limit has been reached. Please see the APM Server docs for 'aggregation.transaction.max_groups' to increase this",
```

Comment:
We already have a similar warning when (another) limit is reached:

https://github.com/elastic/kibana/blob/ae5594849c26775e5a6207258f5fe5139bd3e5a0/x-pack/plugins/apm/public/components/shared/transactions_table/index.tsx#L323-L332

I think showing the warning is still fine, but we shouldn't have two competing limits, or logic for showing warnings. The other warning is showed when `sum_other_doc_count > 0`:

https://github.com/elastic/kibana/blob/2847940b44e6e0de40831dcbf4decbb75c4fac24/x-pack/plugins/apm/server/routes/services/get_service_transaction_groups.ts#L149-L151

Reply:
We should align the behaviour with the changes in: https://github.com/elastic/kibana/pull/148641

Reply:
@sqren Will it be possible to get rid of the whole setting (`xpack.apm.ui.transactionGroupBucketSize`)? I think the limit will be now in the `apm-server` and we will get as many results as that limit allows.

What do you think?

Reply:
>  I think the limit will be now in the apm-server and we will get as many results as that limit allows. 

I think it is a good idea to remove the `xpack.apm.ui.transactionGroupBucketSize` option. It should not be possible to configure a limit in apm server and separately in apm ui.

That being said, when you remove `xpack.apm.ui.transactionGroupBucketSize`  you still need to specify a limit in the terms agg:

https://github.com/elastic/kibana/blob/645a9aa2d35111558e5bc561ad875fb34c3bdc7b/x-pack/plugins/apm/server/routes/services/get_service_transaction_groups.ts#L93-L98

For https://github.com/elastic/kibana/pull/148641 @achyutjhunjhunwala and I decided to set a hardcoded limit of 10.000.

Reply:
@yngrdyn @achyutjhunjhunwala You should reach out to APM Server and mention this to them. Perhaps they can specify 10000 as a hardcoded limit for `aggregation.transaction.max_groups`  and `aggregation.service.max_groups`.

Reply:
@sqren May be i am missing some context here, but if. i look at the [Pitch](https://github.com/elastic/apm-dev/issues/893), i see this limit is decided based on Memory

<img width="600" alt="image" src="https://user-images.githubusercontent.com/7416358/212074261-fa4674d7-f02a-42ed-a572-133af20acdb4.png">


Or am i missing something (may be i am a bit confused)?

Reply:
No, that's correct. I had forgotten about the dynamic limit. But there's still a max: 30 000 groups per service. And 30 000 services. So instead of hardcoding 10 000 we can hardcode 30 000. Although, that seems excessively high to me. I'm not sure about the perf implications of creating 30 000 buckets - let alone showing that in the UI. 

I'm still leaning towards setting a limit of 10 000 in the UI and if that limit is exceeded (`response.aggregations?.transaction_groups.sum_other_doc_count > 0`) we'll show a warning in the UI:

"There are too many unique {transaction groups | services}. Please use the search bar to limit your search"

Again, you should talk to APM Server folks about what they think.

Reply:
As discussed in our meeting we'll set up this limit to `1000` and will be a hard coded value